lib/pages/post_widgets/post_widget.dart

```diff
@@ -67,24 +74,56 @@ class PostWidget extends StatelessWidget {
                         children: [
                           Text(
                             "c/${post.communityName}",
-                            style: const TextStyle(
-                            fontSize: 16),
+                            style: const TextStyle(fontSize: 16),
                             textAlign: TextAlign.left,
                           ),
                           Text(
                             "u/${post.creator}",
                             style: const TextStyle(
-                            fontSize: 24, fontWeight: FontWeight.bold),
+                                fontSize: 24, fontWeight: FontWeight.bold),
                             textAlign: TextAlign.left,
                           ),
                           Text(
                             getFormattedDate(post.creationDate),
                             style: const TextStyle(fontSize: 12),
-                            textAlign: TextAlign.center,
+                            textAlign: TextAlign.left,
                           ),
                         ],
                       ),
                     ),
+                    PopupMenuButton(itemBuilder: (context) {
+                      return [
+                        const PopupMenuItem<int>(
+                          value: 0,
+                          child: Text("Delete"),
+                        ),
+                      ];
+                    }, onSelected: (value) {
+                      if (value == 0) {
```

Comment:
1. can you format this properly, something like:
```
PopupMenuButton(
    itemBuilder: (context) {
            ...
        },
    onSelected: (value) {
            ...
        }
),
```

2. This now shows the "delete" option for everyone? even if the user is not the author of the post?